cfgov/v1/forms.py

```diff
@@ -320,50 +319,36 @@ def get_query_strings(self):
 
 
 class NewsroomFilterForm(FilterableListForm):
-
-    def get_pages(self, parent, hostname):
-        pages = CFGOVPage.objects.live_shared(hostname).descendant_of(parent)
-        blogs = []
+    def __init__(self, *args, **kwargs):
+        parent = kwargs.pop('parent')
+        hostname = kwargs.pop('hostname')
```

Comment:
Well the super call is on FilterableListForm, not NewsroomFilterForm which means it won't ingest a `hostname`.